src/components/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './RadioButton.module.scss';
+
+const RadioButton = ({ children, checked, switchFn }) => (
+	<label className={styles.radio}>
+		<input
+			type="radio"
+			checked={checked}
+			onChange={switchFn}
+		/>
+		<div className={styles.radioButton} />
+		{children}
+	</label>
+);
+
+RadioButton.propTypes = {
+	switchFn: PropTypes.func,
```

Comment:
Try to keep the name of the most common props consistent. In this case, `onChange` is more widely used and everyone would understand its purpose.